torchtune/modules/common_utils.py

```diff
@@ -48,3 +52,81 @@ def reparametrize_as_dtype_state_dict_post_hook(
             state_dict[k] = v.to(dtype)
             if offload_to_cpu:
                 state_dict[k] = state_dict[k].cpu()
+
+
+# mmap.MAP_SHARED is not supported on Windows but this change targets colab.
+if hasattr(torch.serialization, "skip_data") and not sys.platform == "win32":
```

Comment:
Is the first half of this check just a proxy for a particular torch version? If so maybe better to just directly gate on that (with `torch_version_ge` or something)

Reply:
`torch_version_ge` seems to cause circular import when imported in this file :/

Reply:
Oh yeah we are working to fix that, `__torch_version__` is good too